src/R/steekproeftrekking_nabehandeling.R

```diff
@@ -69,3 +69,15 @@ filter_zichtbaarheid <- function(sample, min_cvvi) {
   sample %>%
     filter(cvvi >= min_cvvi)
 }
+
+output_finaal <- function(files, write_out) {
+  if (write_out) {
+    for (i in seq_along(files)) {
+      name <- names(files)[i]
```

Comment:
Does this work as intended? Maybe try `name <- deparse(substitute(files[[i]]))` if you want to get the object's name (instead of the names associated with the columns of the data.frame). But it's very much possible I am the one that is mistaken because of the way this function works in the targets pipeline.